peppy/project.py

```diff
@@ -4,13 +4,16 @@
 import os
 from collections.abc import Mapping
 from logging import getLogger
+from typing import List, Tuple
 
 import pandas as pd
 from attmap import PathExAttMap
 from pandas.core.common import flatten
 from rich.progress import track
 from ubiquerg import is_url
 
+from peppy.sample import Sample
```

Comment:
AFAIK it is recommended to use absolute imports over relative imports, mostly because of readability (please see eg. [here](https://realpython.com/absolute-vs-relative-python-imports/), or Google style guide [here](https://google.github.io/styleguide/pyguide.html#22-imports))